authprox/README.md

```diff
@@ -0,0 +1,50 @@
+# Authentication Proxy
```

Comment:
It would be nice to have that integrated into the rest of the Cothority's README system with the links. See `calypso/README.md` and others.